.github/workflows/build.yml

```
@@ -22,7 +22,7 @@ jobs:
       uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
 
     - name: Install Conda environment with Micromamba
-      uses: mamba-org/provision-with-micromamba@main
+      uses: mamba-org/setup-micromamba@latest
```

Looks like we should use `v1` as the tag?

```suggestion
      uses: mamba-org/setup-micromamba@v1
```